### PR TITLE
Added SAM OpenUser,OpenAlias

### PR DESCRIPTION
**Pull request by sangdo-lee**
```
Results :

Tests run: 544, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.342s
[INFO] Finished at: Mon Oct 30 14:57:35 EDT 2017
[INFO] Final Memory: 26M/389M
[INFO] ------------------------------------------------------------------------
----------------------------------------------------------------------

```

### CODE REVIEW

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/***************************************************************************
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java]: Wrong copyright header.

I made the same mistake.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        this.handle = handle;
+        this.userRid = userRid;
+        this.desiredAccess = desiredAccess;
+    }
+
+    @Override
+    public void marshal(PacketOutput packetOut)
+        throws IOException {
+
+	packetOut.write(handle.getBytes());
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java]: indentation

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcResponse.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/***************************************************************************
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcResponse.java]: Wrong copyright header.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcResponse.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import com.rapid7.client.dcerpc.mssamr.objects.AliasHandle;
+
+/**
+ *
+ */
+public class SamrOpenLocalGroupRpcResponse extends HandleResponse {
+
+    @Override
+    public AliasHandle getHandle() {
+	AliasHandle handle = new AliasHandle();
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcResponse.java]: indentation.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java (hunk 1 above)
Comment (sangdo-lee) [reply src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java]: yep, I will  update it

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+package com.rapid7.client.dcerpc.mssamr.messages;
+
+import java.io.IOException;
+import com.rapid7.client.dcerpc.io.PacketOutput;
+import com.rapid7.client.dcerpc.messages.RequestCall;
+import com.rapid7.client.dcerpc.mssamr.objects.DomainHandle;
+
+/**
+ *
+ */
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java]: Don't leave as empty, either remove or add java doc.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenUserRequest.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+package com.rapid7.client.dcerpc.mssamr.messages;
+
+import java.io.IOException;
+import com.rapid7.client.dcerpc.io.PacketOutput;
+import com.rapid7.client.dcerpc.messages.RequestCall;
+import com.rapid7.client.dcerpc.mssamr.objects.DomainHandle;
+
+/**
+ *
+ */
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenUserRequest.java]: Don't leave as empty, either remove or add java doc.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenUserResponse.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+ * work including confidential and proprietary information of Rapid7.
+ **************************************************************************/
+package com.rapid7.client.dcerpc.mssamr.messages;
+
+import com.rapid7.client.dcerpc.messages.HandleResponse;
+import com.rapid7.client.dcerpc.mssamr.objects.UserHandle;
+
+/**
+ *
+ */
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenUserResponse.java]: Don't leave as empty, either remove or add java doc.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/objects/AliasHandle.java [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+ * All rights reserved. This material contains unpublished, copyrighted
+ * work including confidential and proprietary information of Rapid7.
+ **************************************************************************/
+package com.rapid7.client.dcerpc.mssamr.objects;
+
+import com.rapid7.client.dcerpc.objects.ContextHandle;
+
+/**
+ *
+ */
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/objects/AliasHandle.java]: Don't leave as empty, either remove or add java doc.

File: src/main/java/com/rapid7/client/dcerpc/mssamr/objects/UserHandle.java [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+ * All rights reserved. This material contains unpublished, copyrighted
+ * work including confidential and proprietary information of Rapid7.
+ **************************************************************************/
+package com.rapid7.client.dcerpc.mssamr.objects;
+
+import com.rapid7.client.dcerpc.objects.ContextHandle;
+
+/**
+ *
+ */
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/objects/UserHandle.java]: Don't leave as empty, either remove or add java doc.

File: src/test/java/com/rapid7/client/dcerpc/mssamr/Test_SamrOpenLocalGroup.java [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * * Neither the name of the copyright holder nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ */
+package com.rapid7.client.dcerpc.mssamr;
+
+import java.io.IOException;
+import org.junit.Test;
+
```
Comment (bli-r7) [inline src/test/java/com/rapid7/client/dcerpc/mssamr/Test_SamrOpenLocalGroup.java]: no space

File: src/test/java/com/rapid7/client/dcerpc/mssamr/Test_SamrOpenUser.java [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+ */
+package com.rapid7.client.dcerpc.mssamr;
+
+import java.io.IOException;
+import org.junit.Test;
+
+import static org.bouncycastle.util.encoders.Hex.toHexString;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.junit.Assert.*;
+
```
Comment (bli-r7) [inline src/test/java/com/rapid7/client/dcerpc/mssamr/Test_SamrOpenUser.java]: no space

File: src/test/java/com/rapid7/client/dcerpc/mssamr/Test_SamrOpenLocalGroup.java [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+import static org.bouncycastle.util.encoders.Hex.toHexString;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
```
Comment (bli-r7) [inline src/test/java/com/rapid7/client/dcerpc/mssamr/Test_SamrOpenLocalGroup.java]: no space

**Pull request merged**